api/PositionOptions.json

```diff
@@ -0,0 +1,540 @@
+{
+  "api": {
+    "PositionOptions": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PositionOptions",
+        "support": {
+          "webview_android": [
+            {
+              "version_added": "47"
+            },
+            {
+              "version_added": "5",
+              "version_removed": "47",
+              "notes": "Secure context not required"
+            }
+          ],
+          "chrome": [
+            {
+              "version_added": "47"
+            },
+            {
+              "version_added": "5",
+              "version_removed": "47",
+              "notes": "Secure context not required"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "47"
+            },
+            {
+              "version_added": "5",
+              "version_removed": "47",
+              "notes": "Secure context not required"
+            }
+          ],
+          "edge": [
+            {
+              "version_added": null
```

Comment:
I don't think we need this (or the array)—the entry with the note can stand alone